hugo/content/about.md

```diff
@@ -0,0 +1,58 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a [free, open source](https://en.wikipedia.org/wiki/Free_and_open-source_software), low latency, high quality voice chat application.
+
+## Source Code & Development
+
+The Source Code and further information can be found in our [GitHub repo](https://github.com/mumble-voip/mumble).
+
+For other Mumble Projects, see [Overview of Projects on GitHub](https://github.com/mumble-voip/).
+
+## Licenses
+
+Mumble is [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software) and licensed under a custom BSD-style [License](https://www.mumble.info/LICENSE).
+
+All Licenses are also listed on GitHub:
+
+- [Mumble Repo (Client & Server)](https://github.com/mumble-voip/mumble/)
+    - [Main Mumble License](https://github.com/mumble-voip/mumble/blob/master/LICENSE)
+
+- [Other Mumble Projects (Overview)](https://github.com/mumble-voip/)
+
+
+# About us
+
+We are a team of voluntary developers - both professional and non-professional - from various countries, including Italy, Germany, USA and more.
+
+## Core Team
+
+- Davide Beatrici [davidebeatrici](https://github.com/davidebeatrici) - Core developer
+- Robert Adam [Krzmbrzl](https://github.com/Krzmbrzl)- Core developer, GitHub Management
+- Nik Reist [ZeroAbility](https://github.com/ZeroAbility) - Release Management
+- Jan Klass [Kissaki](https://github.com/Kissaki) - Community work
+- [Natenom](https://github.com/Natenom) - Testing
+
+## Contributors
+
+We are also supported by contributors from the community.
+You find a [List of Contributors on Github](https://github.com/mumble-voip/mumble/graphs/contributors).
```

Comment:
```suggestion
Without further contributors from within the community, this project would not be possible.
You  can find a [list of contributors on GitHub](https://github.com/mumble-voip/mumble/graphs/contributors).
```

Reply:
I see your intention, but I think we might reformulate that a bit.
Something like: "Contributions from the community are an important part of the project."

I know that's not perfect either, but thats more the direction I would go, for two reasons:
1. language (sounds rather german, no offense)
2. exaggeration (main contributors are from the team and former team; so community is a nice addition, but not like 50% of the project)

Reply:
By the way: Should I also mention honorable former team members or a special thanks?

Reply:
> language (sounds rather german, no offense)

I'm actually pretty sure that this is a fine english sentence :shrug: .

But I agree: Your formulation is probably better as it is more correct :+1: 

> By the way: Should I also mention honorable former team members or a special thanks?

Nah, I wouldn't do that. People will find out about those when looking at the contributor page on GitHub.